src/Microsoft.VisualStudio.ProjectSystem.Managed/Telemetry/ITelemetryService.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.VisualStudio.Telemetry
+{
+    /// <summary>
+    /// An internal telemetry service to report telemetry events.
+    /// </summary>
+    internal interface ITelemetryService
+    {
+        void PostProperty(string eventName, string propertyName, string propertyValue);
+        void PostProperties(string eventName, List<(string propertyName, string propertyValue)> properties);
```

Comment:
>List [](start = 46, length = 4)

Shouldn't this be more general, like IEnumerable since the function is just foreach-ing it and we don't have to convert events if they aren't in a list.